graphbolt/include/graphbolt/csc_sampling_graph.h

```diff
@@ -64,6 +64,26 @@ class CSCSamplingGraph : public torch::CustomClassHolder {
       const torch::optional<torch::Tensor>& node_type_offset,
       const torch::optional<torch::Tensor>& type_per_edge);
 
+  /**
+   * @brief Performs neighbor sampling for a given set of seed nodes taking edge
+   * type into account, where each edge type has a specified pick number.
```

Comment:
What's pick number? I don't see edge type are in the parameter list, how does the method take edge type into account.

Reply:
Pick number is `fanouts`, it has same length as edge types.